README.md

```diff
@@ -31,42 +31,42 @@ Add the Cribl Helm repo.
 helm repo add cribl https://criblio.github.io/helm-charts/
 ```
 
-The following example creates a distributed deployment with two autoscaled worker groups, `pcilogs` and `system-metrics`. It uses an auth token of `ABCDEF01-1234-5678-ABCD-ABCDEF012345`, sets an admin password, and installs our license:
+The following example creates a distributed deployment with two auto-scaled worker groups, `pcilogs` and `system-metrics`. It uses an auth token of `ABCDEF01-1234-5678-ABCD-ABCDEF012345`, sets an admin password, and installs our license. For Workers to communication with the Leader node, both Worker Group deployments reference the Service (`ls-leader-internal`) created by the Leader helm chart deployment.
```

Comment:
"For Workers to communication with the Leader node, both Worker Group deployments reference..." should read more like:

"For Workers to communicate with the Leader Node, both Worker Group deployments must reference..."

Reply:
Should be un-mangled now.